j5/backends/hardware/sr/v4/power_board.py

```diff
@@ -65,6 +65,8 @@ class WriteCommand(NamedTuple):
 CMD_WRITE_ERRORLED = WriteCommand(7)
 CMD_WRITE_PIEZO = WriteCommand(8)
 
+usb._objfinalizer._AutoFinalizedObjectBase._do_finalize_object = lambda x: None
```

Comment:
I agree that this arcane hack is justified, but please can we document why its necessary and why a cleaner approach cannot be used? For the benefit of others reading the code, and for our own future reference.

Reply:
Sure.